fundamentals/impl.tex

```diff
@@ -366,27 +349,6 @@ \subsubsection{Resources and Materials}
 please - including ignoring them.  It is the responsibility of archetype
 developers to choose how to handle potentially full-fidelity compositions.
 
-In large simulations, many material objects may change frequently.
```

Comment:
So, I do think this part is too wordy and needs to be vastly decreased (or accompanied by a figure). However, I think at least the idea in the first sentence is somewhat important.  I'm going to revert this commit, but then add some commits that pare down this section dramatically.